packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -340,14 +347,19 @@ export const createDefaultCallingHandlers = memoizeOne(
     };
 
     /* @conditional-compile-remove(PSTN-calls) */
-    const onAddParticipant = async (
-      participant: CommunicationIdentifier,
-      options?: AddPhoneNumberOptions
-    ): Promise<void> => {
-      if (isPhoneNumberIdentifier(participant)) {
-        await call?.addParticipant(participant, options);
-      } else if (isCommunicationUserIdentifier(participant) || isMicrosoftTeamsUserIdentifier(participant)) {
-        await call?.addParticipant(participant);
+    const onAddParticipant: AddParticipantHandler = async (participant, options?) => {
+      const participantType = participantTypeHelper(participant);
+      switch (participantType) {
+        case 'PSTN':
+          if (options) {
+            await call?.addParticipant(participant as PhoneNumberIdentifier, options);
```

Comment:
Aren't `options` optional in the headless API as well? We shouldn't add another layer of validation. Instead, the headless SDK should throw if it really needs the options to be defined.

Reply:
+1
@dmceachernmsft let's keep this optional in our api as well.
And let the headless sdk throw :)